### PR TITLE
add boostrab 3 submenu template

### PR DESCRIPTION
**Pull request by PetrDlouhy**
PR for #246

### CODE REVIEW

File: sitetree/templates/sitetree/menu_bootstrap3_deep.html~ [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{% load sitetree %}
```
Comment (idlesign) [inline sitetree/templates/sitetree/menu_bootstrap3_deep.html~]: Please remove this backup file.

File: sitetree/static/css/sitetree_bootstrap_submenu.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
+// Taken from https://codepen.io/ajaypatelaj/pen/prHjD
```
Comment (idlesign) [inline sitetree/static/css/sitetree_bootstrap_submenu.css]: This probably should be minified. What do you think?

File: docs/source/templatesmod.rst [hunk 3]
```diff
@@ -151,6 +151,14 @@ The following templates are bundled with SiteTree:
 
    Constructs nav-pills Bootstrap 3 horizontal navigation.
 
+ * `sitetree/menu_bootstrap3_deep.html`
+
+   Constructs Bootstrap 3 menu with infinite submenus.
+   Requires adding extra CSS::
+
+   <link href="/static/css/sitetree_bootstrap_submenu.css" type="text/css" rel="stylesheet" media="screen">
```
Comment (idlesign) [inline docs/source/templatesmod.rst]: ``/static/`` shouldn't be harcoded, since it can change.

Comment (PetrDlouhy) [reply docs/source/templatesmod.rst]: Of course, that is mistake.

File: sitetree/templates/sitetree/menu_bootstrap3_deep.html~ (hunk 1 above)
Comment (PetrDlouhy) [reply sitetree/templates/sitetree/menu_bootstrap3_deep.html~]: Oh, that shouldn't be there, of course.

File: sitetree/static/css/sitetree_bootstrap_submenu.css (hunk 2 above)
Comment (PetrDlouhy) [reply sitetree/static/css/sitetree_bootstrap_submenu.css]: I think, that we should keep the human-editable version of this file. So I can keep this and include also the minified version, although I don't think it is ideal due to code duplicity in the repo.
The ideal solution would be to compress the file in `setup.py` during packing.

Another possibility is to only suggest using `django-compressor` in the docs without dealing with the minification at all.

**Comment by PetrDlouhy** [comment]
@idlesign I fixed the obvious errors and suggested three possibilities for the minification. Pleas choose the best option, and I will update the PR by that.

**Comment by idlesign** [comment]
Unminified will do. Merged.
Thank you.

**Pull request merged**